.github/workflows/node.js.yml

```diff
@@ -8,22 +8,70 @@ on:
   workflow_call:
 
 jobs:
+  
   build:
-
     runs-on: ubuntu-latest
-
+    outputs:
+      coverage: ${{ steps.coverage.outputs.coverage }}
+      color: ${{ steps.coverage.outputs.color }}
     strategy:
       matrix:
         node-version: [16.x, 18.x]
         # See supported Node.js release schedule at https://nodejs.org/en/about/releases/
-
     steps:
     - uses: actions/checkout@v3
     - name: Use Node.js ${{ matrix.node-version }}
       uses: actions/setup-node@v3
       with:
         node-version: ${{ matrix.node-version }}
         cache: 'npm'
+    - name: Checkout code
+      uses: actions/checkout@v2
     - run: npm ci
     - run: npm run fmt:check
-    - run: npm run test:ci
+    - id: coverage
```

Comment:
We will need to redo that for every assemblyscript project. How to centralized that ?

Reply:
I think we can create a repository for a reusable github action

Reply:
For now we could merge it and add issue to make it reusable